src/Widgets/DataFormUploadWidget/DataFormUploadWidgetComponent.tsx

```diff
@@ -87,7 +88,11 @@ provideComponent(DataFormUploadWidget, ({ widget }) => {
         </>
       ) : null}
 
-      <div {...getRootProps({ className: 'dropzone' })}>
+      <div
+        {...getRootProps({
+          className: `dropzone ${isDragActive ? 'border border-3 rounded-3' : ''}`,
```

Comment:
* [ ] The larger border causes layout shifts on hovering. In the worst case this may make dropping impossible. How about reusing our style for `.form-control:focus`?